lisa/operating_system.py

```diff
@@ -487,6 +487,9 @@ def add_azure_core_repo(
     def set_kill_user_processes(self) -> None:
         raise NotImplementedError("set_kill_user_processes is not implemented")
 
+    def add_key(self, server_name: str, key: str) -> None:
```

Comment:
Is it a general method for all distro?

Reply:
I know for fedora, SUSE, they use `rpm --import key`,

Reply:
Let's keep it in Debian only, until we implement another os, and find the common pattern.